sugoi-api-core/src/main/java/fr/insee/sugoi/core/store/WriterStore.java

```diff
@@ -13,11 +13,30 @@
 */
 package fr.insee.sugoi.core.store;
 
+import fr.insee.sugoi.model.Group;
 import fr.insee.sugoi.model.User;
 
 public interface WriterStore {
 
-  String deleteUser(String domain, String id);
+  void deleteUser(String id);
 
   User createUser(User user);
+
+  User updateUser(User updatedUser);
```

Comment:
I chose not to add specific methods for specific changes (like operations on user credentials) and rather always rely on updating the entire object (apart for the group methods)

Reply:
I'll add reinitCredential and initCredential but does the credential init status should be in the User model ?

Reply:
Not sure, but this could be added later.

We also need a sort of validateCredential method (to verify credential)

Reply:
oups, I did not see it in ReaderStore interface, you can forget the last remark.